target/linux/realtek/rtl930x/config-6.12

```diff
@@ -123,7 +123,7 @@ CONFIG_MIPS=y
 CONFIG_MIPS_ASID_BITS=8
 CONFIG_MIPS_ASID_SHIFT=0
 # CONFIG_MIPS_CMDLINE_FROM_BOOTLOADER is not set
-CONFIG_MIPS_CMDLINE_FROM_DTB=y
+CONFIG_MIPS_CMDLINE_DTB_EXTEND=y
```

Comment:
On ath79, this was done by simply removing the bootargs on devices which got this information from u-boot:

```
	chosen {
		/delete-property/ bootargs;
	};
```

have to check what/if there was a problem with realtek MIPS

Reply:
How would that work? /delete-property/ removes a previously defined property at device-tree-compile-time.

Reply:
> How would that work? /delete-property/ removes a previously defined property at device-tree-compile-time.

1. it works
2. @Rondom  `CONFIG_MIPS_CMDLINE_FROM_DTB` just just enables code which overwrites the cmdline with the content of `bootargs`. if it is not present, it will simply not overwrite anything. You can find the code which implements this behavior at https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git/tree/arch/mips/kernel/setup.c?id=37816488247ddddbc3de113c78c83572274b1e2e#n588